src/components/games/index.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
+import lebron from 'assets/gifs/lebron.gif';
```

Comment:
rename
```
import gamesGif from 'assets/gifs/games.gif';
```

where this should be the gif: https://media3.giphy.com/media/5n1XqQh0DskgKMKejY/giphy.gif